lib/takeaway.rb

```diff
@@ -0,0 +1,67 @@
+require_relative "menu"
+require_relative "sms"
+
+class Takeaway
+
+  def initialize(menu: Menu.new,sms: SMS.new)
+    @menu_class = menu
+    @sms = sms
+    @selected_dishes = Hash.new(0)
+  end
+
+  def list_dishes
+    @menu_class.dishes
+  end
+
+  def order
+    while true
+      numbered_list_dishes
+      input = user_input
+      break if input == 99
```

Comment:
So, 99 is like the esc. key right? And you have the menu ordered by number... It's a smart and neat way to do it!